### ISSUE TITLE
Add GTL001

### ISSUE THREAD

**Issue by eliheuer** [google/fonts#6002]
https://github.com/eliheuer/GTL001

GTL001 is the first typeface released by GTL Type Label and was drawn by [elih.eth](https://rainbow.me/elih.eth)(Eli Heuer). The design takes inspiration from happy hardcore music graphics and 90's rave fliers. A 100/100 NFT for this typeface was minted on [Zora](https://zora.co/collections/0xb013e4f218760bb72f63f8d78a82b546c91897ef), and an essay titled “[Libre Type as a Veblen Good](https://www.nan.xyz/txt/libre-type-as-a-veblen-good/)” was written about the project.

I understand that Google Fonts will publish only fonts that matches its requirements, and I can confirm the project meets them (by ticking the cases, or putting x between the square brackets in text mode):

- [x] The entire font project is licensed under the [OFL](https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL)
- [x] There is no "Reserved Font Name" mention in the OFL license
- [x] Family name is unique according to namecheck.fontdata.com
- [x] The font supports at least [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/)
- [x] The repository respect the [upstream repo structure](https://googlefonts.github.io/gf-guide/upstream.html)
- [x] The source files are available in the repository
- [x] I read, agree and comply with the full [contributing requirements](https://googlefonts.github.io/gf-guide/onboarding.html)
- [x] I am the author of the font or the author agreed to this submission
- [x] I will maintain the repository

![Alpha-NFT](https://user-images.githubusercontent.com/5162664/224280323-1b000fe6-7b22-44eb-a669-53f9903b99b7.png)

![zora](https://user-images.githubusercontent.com/5162664/224280409-6e0f7398-effa-49a0-9fd1-60f6c796ea85.jpeg)

<img width="1637" alt="gtl001-test" src="https://user-images.githubusercontent.com/5162664/224280697-36e8dc1f-c1ad-4d94-a31b-85281d26cfe3.png">

**Comment by RosaWagner** [comment]
Hey @eliheuer super cool work, I added it to the submission to review. You know that GF is not publishing fonts with such small latin glyphset though… do you plan on completing the Google latin Kernel or the Latin Core?

**Comment by eliheuer** [comment]
@RosaWagner Yes, I should not have checked the glyphset box yet, sorry. What I meant was that I plan on getting this to [GF Latin Core glyphset](https://github.com/googlefonts/glyphsets/tree/main/GF_glyphsets/Latin/) soon.

I unchecked the glyphset box and will recheck and post here when I'm done.

**Comment by eliheuer** [comment]
I should note that this typeface is similar to Baby Teeth designed by Milton Glaser in 1964: https://fontsinuse.com/typefaces/7322/baby-teeth

But if you look at them together you will see my design is not simply a tracing of that font and has completely unique proportions and features. Milton Glaser himself copied the style from a hand-painted sign in Mexico City.

![FcF1mG3X0AEvmyZ](https://user-images.githubusercontent.com/5162664/224298996-2baf0579-2afa-4ea2-89ea-42eb8ef12989.jpeg)
![tumblr_perjrjG9VC1s9jvclo1_1280](https://user-images.githubusercontent.com/5162664/224299008-cf7f69ed-a1c3-4b88-a14f-d7e714dac983.jpeg)
